ossapi/ossapiv2.py

```diff
@@ -2665,6 +2666,28 @@ def download_score_mode(
         url = f"{self.base_url}/scores/{mode.value}/{score_id}/download"
         return self._download_score(url=url, raw=raw)
 
+    @request(Scope.PUBLIC, category="scores")
+    def scores(
+        self, mode: GameModeT, *, cursor_string: Optional[str] = None
+    ) -> Scores:
```

Comment:
let's move this to right after `def score` (I do try to keep the endpoints in `Ossapi` in alphabetical order)